src/main/java/dan200/computercraft/core/apis/OSAPI.java

```diff
@@ -271,6 +370,22 @@ public final int day( Optional<String> args ) throws LuaException
         }
     }
 
+    /**
+     * Returns the number of seconds since an epoch depending on the locale.
+     *
+     * * If called with {@code ingame}, returns the number of seconds since the
+     *   world was created. This is the default.
+     * * If called with {@code utc}, returns the number of seconds since 1
+     *   January 1970 in the UTC timezone.
+     * * If called with {@code local}, returns the number of seconds since 1
+     *   January 1970 in the server's local timezone.
+     *
+     * @param args The locale to get the seconds for. Defaults to {@code ingame} if not set.
+     * @return The seconds since the epoch depending on the selected locale.
+     * @cc.tparam [opt] string The locale to get the seconds for. Defaults to {@code ingame} if not set.
+     * @cc.treturn number The seconds since the epoch depending on the selected locale.
```

Comment:
See comments at the top about `[opt]`. I don't _think_ you need the explicit `cc.treturn` either here though, as the function explicitly returns a `long`.